gnoi/reset/client_test.go

```diff
@@ -0,0 +1,82 @@
+/* Copyright 2020 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package reset
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/google/gnxi/gnoi/reset/pb"
+)
+
+func TestCheckResponse(t *testing.T) {
+	tests := []struct {
```

Comment:
missing:

```
response: &pb.StartResponse{&pb. StartResponse_ResetSuccess{&pb.Success{}}},
want: nil,
```

Reply:
I've included this in the tests now, I ran into some segmentation fail issues with cmp.Diff trying to compare nil versus nil, so I've used a continue in the loop to resolve this

Reply:
if you return *ResetError here I think you wont have the issue:

func CheckResponse(res *pb.StartResponse) *ResetError {
}

would you try?

Reply:
That worked, thank you!